README.md

```diff
@@ -267,13 +268,39 @@ At the moment, you can ignore
 * `filesize`
 
 
+## Checksum Calculator
+
+Talisman Checksum calculator gives out yaml format which you can directly copy and paste in .talismanrc file in order to ignore particular file formats from talisman detectors.
+
+To run the checksum please "cd" into the root of your repository and run the following command
+
+For Example:
+`talisman --checksum="*.pem *.txt"`
+
+1. This command finds all the .pem files in the respository and calculates collective checksum of all those files and outputs a yaml format for .talismanrc. In the same way it deals with the .txt files.
+2. Multiple file names / patterns can be given with space seperation.
+
+Example output:
+```
+.talismanrc format for given file names / patterns
+fileignoreconfig:
+- filename: '*.pem'
+  checksum: f731b26be086fd2647c40801630e2219ef207cb1aacc02f9bf0559a75c0855a4
+  ignore_detectors: []
+- filename: '*.txt'
+  checksum: d9e9e94868d7de5b2a0706b8d38d0f79730839e0eb4de4e9a2a5a014c7c43f35
+  ignore_detectors: []
+```
+
+Note: Checksum calculator considers the staged files while calculating the collective checksum of the files.
+
 ## Scanning Git history
 
 Talisman also scans the content present in the git history of the repository, this includes scanning of the files listed in the .talismanrc file as well.
 
 To run the scanner please "cd" into the directory to be scanned and run the following command
 
-* `talisman scan`
+* `talisman --scan`
```

Comment:
Multiple PR's are trying to fix this line. Also looks like an unrelated change.
Please consider dropping this change.

Reply:
reverted back.